src/controller/CHIPDeviceController.cpp

```diff
@@ -416,6 +417,14 @@ CHIP_ERROR DeviceCommissioner::Init(CommissionerInitParams params)
 
     mPairingDelegate = params.pairingDelegate;
 
+    mICDRegistrationDelegate = params.icdRegistrationDelegate;
+    if (mICDRegistrationDelegate == nullptr)
+    {
+        ChipLogProgress(
+            Controller,
+            "*** Missing ICDRegistrationDelegate at DeviceCommissioner init: the controller cannot commission an ICD device!");
```

Comment:
This log message is wrong. It can't commission a LIT as a LIT, but it can commission a LIT to operate as a SID, and it can commission SITs just fine.  Please fix this to not be misleading.